README.md

```diff
@@ -1,117 +1,135 @@
-Using publisher: Dashboard Publisher
-Fetched 3 definitions
-Fetched 1 policies
-Processing APIs...
-Deleting: 0
-Updating: 3
-Creating: 0
-SYNC Updating: 598ec94f9695f201730d835b
-SYNC Updating: 598ec9589695f201730d835c
-SYNC Updating: 5990cfee9695f201730d836e
-Processing Policies...
-Deleting policies: 0
-Updating policies: 1
-Creating policies: 0
-SYNC Updating Policy: Test policy 1
---> Found policy using explicit ID, substituting remote ID for update
-```
-
-The command provides output to identify which actions have been taken. If using a Tyk Gateway, the Gateway will be
-automatically hot-reloaded.
-
+# Tyk-Sync
+
+## What is it?
+
+Tyk-Sync is a command line tool and library to manage and synchronise a Tyk installation with your version control system (VCS).
+
+## Features
+
+- Update APIs and policies on remote Tyk Dashboards
+- Update APIs on remote Tyk CE Gateways
+- Publish APIS/Policies to remote Tyk Dashboards
+- Publish APIs to remote Tyk CE Gateways
+- Synchronise a Tyk Dashboard's APIs and Policies with your VCS (one-way, definitions are written to the Dashboard)
+- Synchronise a Tyk CE Gateway's APIs with those stored in a VCS (one-way, definitions are written to the Gateway)
+- Dump Policies and APIs in a transportable format from a Dashboard to a directory
+- Support for importing, converting and publishing Swagger (Open API Spec) files to Tyk.
+- Specialized support for Git. But since API and policy definitions can be read directly from
+the file system, it will integrate with any VCS.
+
+### Sync
+
+Tyk-Sync tries to be clever about what APIs and Policies to update and which to create, it will actually base all
+ID matching on the API ID and the masked Policy ID, so it can identify the same object across installations. Tyk has
+a tendency to generate fresh IDs for all new Objects, so Tyk-Sync gets around this by using portable IDs and ensuring
+the necessary portable IDs are set when using the `dump` command.
+
+This means that Tyk-Sync can be used to back-up your most important API Gateway configurations as code, and to deploy
+those configurations to any target and ensure that API IDs and Policy IDs will remain consistent, ensuring that any
+dependent tokens continue to have access to your services.
+
+### Prerequisites:
+
+- Tyk-Sync was built using Go 1.10. The minimum Go version required to install is 1.7.
+- In order for policy ID matching to work correctly, your Gateway must have `policies.allow_explicit_policy_id: true`.
+- It is assumed you have a Tyk CE or Tyk Pro installation.
+
+## Installation
+
+Currently the application is available via Go, Docker and in packagecloud, so to install via Go you must have Go installed and run:
+
+ ```
+ go install -i github.com/TykTechnologies/tyk-sync
+ ```
+This should make the `tyk-sync` command available to your console.
+
+### Docker:
+
+To install the latest version of `tyk-sync` via docker image please run:
+```
+docker pull tykio/tyk-sync
+```
+To run `tyk-sync` do:
```

Comment:
I think it's more useful to run it as a one-off, e.g. `docker run -it --rm tyk-sync:latest ./tyk-sync help`, since it's a command rather than a long-running process. Also see the commend in the dockerfile to simplify it a bit more.

Reply:
Readme is now updated.